PhotDM.tex

```diff
@@ -514,13 +516,18 @@ \section{Photometry Data Model} \label{datamodel}
  %%%%%%%%%%%%  Starting New Page here %%%%%%%%%%%%%%
 
 \newpage
+%In order to fully describe values of the magnitudes inside photometry point instances, the class diagram makes use of physical quantity classes. 
+%These classes glue all the basic fields that compose a physical measurement: value, error, units, etc. However, within the present specification, we will describe individual attributes of the different quantities and as a consequence. All the utypes will be also generated from individual physical quantity attributes what will facilitate the use within IVOA Data Access Layer protocols.
+
+proposal Mireille: 
```

Comment:
Please don't comment out text in version-controlled documents: the originals remain in the history anyway, and nobody dares remove the comments any more.  Also, don't write "proposal <name>": That's in the VCS metadata (this comes in a PR from you, after all), and there's no risk of "proposal Mireille" getting merged into the normative text.  Anyway: this has to go before the PR can be merged.